src/frontend/src/pages/HomePage/components/GuestOrganizationInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import { Card, Typography, useTheme } from '@mui/material';
+import { Grid } from '@mui/material';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import React from 'react';
+import { NERButton } from '../../../components/NERButton';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+import { useAllLinkTypes, useAllUsefulLinks } from '../../../hooks/projects.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { Stack } from '@mui/system';
+
+interface GuestOrganizationInfoButtonProps {
+  href?: string;
+  buttonText: string;
+  iconName: string;
+}
+
+const NERGuestButton: React.FC<GuestOrganizationInfoButtonProps> = ({ href, buttonText }) => {
```

Comment:
You currently have the icon name in the interface, but are not using it in the component. Make sure you pass it into the component and use the given Icon instead of the InfoRoundedIcon